api/app/Traits/Generator/GeneratesUserDoc.php

```diff
@@ -0,0 +1,418 @@
+        }
+
+        $this->addLabelText($section, $this->localizeHeading('priority_entitlement'), $this->yesOrNo($user->has_priority_entitlement));
+
+        if ($user->has_priority_entitlement) {
+            $this->addLabelText($section, $this->localizeHeading('priority_number'), $user->priority_number);
+        }
+    }
+
+    /**
+     * Generate information about a users work location
+     *
+     * @param  Section  $section  The section to add info to
+     * @param  User  $user  The user being generated
+     */
+    protected function workLocation(Section $section, User $user, $headingRank = 4)
+    {
+        $section->addTitle($this->localizeHeading('work_location'), $headingRank);
+        $this->addLabelText($section, $this->localizeHeading('work_location'), $this->localizeEnumArray($user->location_preferences, WorkRegion::class));
+        $this->addLabelText($section, $this->localizeHeading('location_exemptions'), $user->location_exemptions ?? '');
+    }
+
+    /**
+     * Generate information about a users work preferences
+     *
+     * @param  Section  $section  The section to add info to
+     * @param  User  $user  The user being generated
+     */
+    protected function workPreferences(Section $section, User $user, $headingRank = 4)
+    {
+        $section->addTitle($this->localizeHeading('work_preferences'), $headingRank);
+        foreach ($user?->position_duration ?? [] as $duration) {
+            $section->addListItem($this->localizeEnum($duration, PositionDuration::class));
+        }
+
+        $preferences = $user->getOperationalRequirements();
+        if (count($preferences['accepted']) > 0) {
+            $section->addText($this->localizeHeading('accepted_operational_requirements'), $this->strong);
+
+            foreach ($preferences['accepted'] as $preference) {
+                $section->addListItem($this->localizeEnum($preference, OperationalRequirement::class, 'long'));
+            }
+        }
+
+        if (count($preferences['not_accepted']) > 0) {
+            $section->addText($this->localizeHeading('rejected_operational_requirements'), $this->strong);
+
+            foreach ($preferences['not_accepted'] as $preference) {
+                $section->addListItem($this->localizeEnum($preference, OperationalRequirement::class, 'long'));
+            }
+        }
+    }
+
+    /**
+     * Generate Diversity, equity and inclusion info for  auser
+     *
+     * @param  Section  $section  The section to add info to
+     * @param  User  $user  The user being generated
+     */
+    protected function dei(Section $section, User $user, $headingRank = 4)
+    {
+        $section->addTitle($this->localizeHeading('dei'), $headingRank);
+
+        if ($user->indigenous_communities) {
+            foreach ($user->indigenous_communities as $community) {
+                $section->addListItem($this->localizeEnum($community, IndigenousCommunity::class, 'long'));
+            }
+        }
```

Comment:
Something wrong here I think

![image](https://github.com/user-attachments/assets/fdab2476-6d8c-4dc0-97e4-e74d78ebfe9e)

Reply:
Fixed in [fix headings, experience spacing](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11151/commits/9ea766020d8987729f9b84b5050c074d5a5aa8db)

Reply:
The other ones work

![image](https://github.com/user-attachments/assets/4ad2856a-7400-42cc-8fc9-453545d0366f)

But not this one, a problem? 🤔 

![image](https://github.com/user-attachments/assets/a36dd691-19c7-4239-886f-12544d322e89)

Reply:
Looks like we actually are not rendering the legacy community on the current doc so it should be filtered out after [do not render legacy indigenous community](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11151/commits/b12b6802a4cc6fd00457ad8ca7aca40ca93a31e1)